fe/fe-core/src/main/java/com/starrocks/task/ExportPendingTask.java

```diff
@@ -117,6 +117,7 @@ private Status makeSnapshots() {
                 if (!GlobalStateMgr.getCurrentSystemInfo().checkBackendAvailable(backendId)) {
                     return Status.CANCELLED;
                 }
+                this.job.setBeStartTime(backendId, backend.getLastStartTime());
```

Comment:
Is it enough to update the be's start time when making snapshot? Be may be restarted after the task is sent to be.